### PR TITLE
fix apache kafka broken link #3159

### PR DESCRIPTION
**Pull request by zhaojizhuang**
fix apache kafka broken link #3159

### CODE REVIEW

**Comment by zhaojizhuang** [comment]
cc @pierDipi

**Review by pierDipi — APPROVED** [review]
/lgtm

**Comment by pierDipi** [comment]
Thank you so much!

File: docs/eventing/sources/_index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ kn source list
...
 | [Apache CouchDB](https://github.com/knative-sandbox/eventing-couchdb/blob/master/source)                                                   | v1alpha1 | Knative    | Brings [Apache CouchDB](https://couchdb.apache.org/) messages into Knative.  |
-| [Apache Kafka](https://github.com/knative/eventing-contrib/blob/master/kafka/source/pkg/apis/sources/v1alpha1/kafka_types.go)       | v1beta1  | Knative    | Brings [Apache Kafka](https://kafka.apache.org/) messages into Knative. The KafkaSource reads events from an Apache Kafka Cluster, and passes these events to a sink so that they can be consumed. See the [Kafka Source](https://github.com/knative/eventing-contrib/tree/master/kafka/source) example for more details.  |
+| [Apache Kafka](https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/apis/sources/v1alpha1/kafka_types.go)       | v1beta1  | Knative    | Brings [Apache Kafka](https://kafka.apache.org/) messages into Knative. The KafkaSource reads events from an Apache Kafka Cluster, and passes these events to a sink so that they can be consumed. See the [Kafka Source](https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/source) example for more details.  |
```
Comment (vaikas) [inline docs/eventing/sources/_index.md:41]: Should this be pointing to:
.../pkg/apis/sources/v1beta1/kafka_types.go

instead of v1alpha1?

Comment (zhaojizhuang) [reply docs/eventing/sources/_index.md:41]: yeah,that looks more appropriate when pointing to b1beta1

**Comment by vaikas** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**